mitiq/interface/mitiq_qiskit/tests/test_qiskit_utils.py

```diff
@@ -174,3 +180,136 @@ def test_circuit_is_not_mutated_by_executors():
     )
     assert single_qubit_circ.data == expected_circuit.data
     assert single_qubit_circ == expected_circuit
+
+
+def test_sample_bitstrings():
+    """Tests that the function sample_bitstrings returns a valid
+    mitiq.MeasurementResult.
+    """
+
+    two_qubit_circ = QuantumCircuit(2, 1)
+    two_qubit_circ.cx(0, 1)
+    two_qubit_circ.measure(0, 0)
+
+    measurement_result = sample_bitstrings(
+        circuit=two_qubit_circ,
+        backend=None,
+        noise_model=initialized_depolarizing_noise(0),
+        shots=5,
+    )
+    assert measurement_result.result == [[0], [0], [0], [0], [0]]
+    assert measurement_result.qubit_indices == (0,)
+
+    two_qubit_circ = QuantumCircuit(2)
+    two_qubit_circ.cx(0, 1)
+    measurement_result = sample_bitstrings(
+        circuit=two_qubit_circ,
+        backend=None,
+        noise_model=initialized_depolarizing_noise(0),
+        shots=2,
+        measure_all=True,
+    )
+    assert measurement_result.result == [[0, 0], [0, 0]]
+    assert measurement_result.qubit_indices == (0, 1)
+    assert isinstance(measurement_result, MeasurementResult)
```

Comment:
Is this standard to have what kind of seems like 2 tests in a single function? (First test checks the return value is of the form we expect, and the second ensures `measure_all` works as expected)

Reply:
I guess it's always better to have small tests. I was too lazy. Test splitted.